mtgjson5/set_builder.py

```diff
@@ -418,6 +423,19 @@ def build_mtgjson_set(set_code: str) -> Optional[MtgjsonSetObject]:
     return mtgjson_set
 
 
+def build_foreign_printed_languages(card_data: List[MtgjsonCardObject]) -> List[str]:
+    """
+    Constructs list of non-english languages that a set is printed in
+    :param card_data: all cards from that set
+    :return: a list of all non-english languages that a set is printed in
+    """
+    language_set: Set[str] = set()
+    for card in card_data:
```

Comment:
The variable names in this function are a bit iffy imo, it's hard to follow.

I'd recommend taking "cards" as your arg, and "entry" instead of lang